### DISCUSSION TITLE
After successfully setting up the master data folder, the "mp" folder can be accessed in the frontend. However, there is an issue with the variable name "TENANT_TENANTS_MP" that is being displayed. To resolve this, consider adjusting the naming convention for better clarity and consistency in the code.

### DISCUSSION THREAD

**Discussion by Amitsinghongraph** [egovernments/Digit-Core#272] [Announcements]
Now I am Facing issue naming convention variable name issue like TENANT_TENANTS_MP and city name TENANT_TENANTS_GWALIOR so how can we resolve this issue . please help community team

**Comment by jagankumar-egov** [comment]
For every code that is added to the tenant's master, the frontend will display a new code. 
It is necessary to map code messages to support different languages.

It's necessary to add a code message pair to the localization service.
a sample pair of keys and values 
```
{
        "code": "TENANT_TENANTS_MP",
        "message": "Madhya Pradesh",  // as per requirement
        "module": "rainmaker-common",
        "locale": "en_IN"
    }
```
To learn more about it, refer to the document below.
https://core.digit.org/platform/core-services/localization-service#steps-to-integration

The prefix is <unk> TENANT_TENANTS_<unk> because the master name is tenant and the module name is tenants.


Clear the browser cache and check again after adding localization.
